inception-pdf-editor/src/main/js/pdfanno/src/pdfanno.js

```diff
@@ -159,6 +160,14 @@ async function displayViewer () {
             window.annoPage.scrollToAnnotation(moveTo)
           }, 500)
         }
+*/
+        parent.Wicket.Ajax.ajax({
+          "m" : "GET",
+          "u" : annoURL,
+          "sh" : [],
+          "fh": []
```

Comment:
I guess it would be a good idea to display an error message if there was a failure - at least to log one to the JS console.